### ISSUE TITLE
F401 imported but unused

### ISSUE THREAD

**Issue by busunkim96** [googleapis/gapic-generator-python#324]
More unused imports (but different from the ones in #169).

If the answer is "It's hard to get rid of these" I can live with that, but I would still like to track this in an issue. 

```
google/cloud/billing_v1/services/cloud_catalog/client.py:19:1: F401 'typing.Iterable' imported but unused
google/cloud/billing_v1/services/cloud_catalog/client.py:19:1: F401 'typing.Iterator' imported but unused
google/cloud/billing_v1/services/cloud_catalog/client.py:23:1: F401 'google.api_core.exceptions' imported but unused
google/cloud/billing_v1/services/cloud_billing/client.py:19:1: F401 'typing.Iterable' imported but unused
google/cloud/billing_v1/services/cloud_billing/client.py:19:1: F401 'typing.Iterator' imported but unused
google/cloud/billing_v1/services/cloud_billing/client.py:23:1: F401 'google.api_core.exceptions' imported but unused
tests/unit/billing_v1/test_cloud_catalog.py:21:1: F401 'math' imported but unused
tests/unit/billing_v1/test_cloud_billing.py:21:1: F401 'math' imported but unused
```

**Comment by software-dov** [comment]
Mostly fixed by #329 
Unused 'math' import is still present in generated tests.

**Comment by larkee** [comment]
For Cloud Spanner, `tests/unit/gapic/spanner_v1/test_spanner.py` contains the following unused imports:
```
from google.cloud.spanner_v1.types import type
from google.cloud.spanner_v1.types import type as gs_type
```

The first import is causing test breakages as it clashes with the type() builtin.

**Comment by software-dov** [comment]
Fixed by #595
